sync_settings/sync_version.py

```diff
@@ -37,7 +38,7 @@ def get_remote_version():
 
 def update_config_file(info):
     with open(file_path, 'w') as f:
-        json.dump(info, f)
+        f.write(sublime.encode_value(info, True))
```

Comment:
add unit tests to ensure that all changes are solving the issues that you are mentioning.

Reply:
👍